CHANGES/8857.feature

```diff
@@ -0,0 +1 @@
+Added ``excludes`` parameter to sync call to perform synclist filtering
```

Comment:
Can this become a `.misc` release note instead of `.feature`. The 0.7.z shouldn't have any new features declared.